### DISCUSSION TITLE
Action strip row context

### DISCUSSION THREAD

**Discussion by Eralmidia** [IgniteUI/igniteui-angular#10646] [Q&A]
## Question  
I'm trying to implement an action strip for a grid. The actions will vary based on the row I'm hovering (which action are visible/disabled etc). How do make the row data available for the action strip in order to evaluate what to show? Since action strip is shown on hover, I can't read the selected row, and there doesn't seem to be a hover event for the grid? I saw the context input for the strip, but I don't understand how that is suppose to work (or if that is actually something I can use to achieve this)

Btw, do you prefer questions to be opened as question in issues, or directly in the discussions tab?

 * igniteui-angular version: 
 * browser:

**Comment by ChronosSF** [comment]
We are trying to move all questions to the Discussions tab but don't worry about where you are posting it. We'll probably have some automation around it soon as well.

**Comment by ChronosSF** [comment]
The Custom Implementation part of the Row Actions topic should provide the guidance on how to use the action strip in your use-case - https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/row-actions#custom-implementation
